scripts/main.py

```diff
@@ -268,7 +268,7 @@ def writer(infer, intervals_file, exp_dir, result_fname,
             # Only one file should be left after the combiner stage
             assert (len(files) == 1)
             # move final files out of tmp folder
-            shutil.move(files[0], outfiles[channel])
+            shutil.copy(files[0], outfiles[channel])
```

Comment:
@ntadimeti should there be an `os.remove` here as well?

Reply:
we have a os.rmtree right after the loop. Didn't find it necessary to have os.remove explicitly ?

Reply:
Cool, I saw the rmtree on the tempdir but wasn't sure if it removed the same folder as the move operation. Looks like it does! Resolving.